core/src/apps/management/apply_settings.py

```diff
@@ -153,8 +153,9 @@ async def _require_confirm_change_label(ctx: GenericContext, label: str) -> None
         ctx,
         "set_label",
         "Device name",
-        description="Do you want to change device name to {}?",
-        description_param=label,
+        description="Do you want to change device name to",
```

Comment:
Change it to "Change device name to…" please.

Reply:
bf3adc743e81ec665638b9edd57a1981395279b6:

![Screenshot 2023-05-29 at 13-14-07 TT-device_tests-test_msg_applysettings py test_apply_settings](https://github.com/trezor/trezor-firmware/assets/85857/865506ce-8999-443e-b5cf-0fb4687326ff)